src/js/common/schemaform/FormPage.jsx

```diff
@@ -102,6 +104,18 @@ class FormPage extends React.Component {
     return { pages: expandedPageList, pageIndex };
   }
 
+  autoSave() {
+    const { route, params, form } = this.props;
+    let data = form.data;
+    const { formId, version } = form;
+    const returnUrl = this.props.locationPathname;
+
+    if (route.pageConfig.showPagePerItem) {
```

Comment:
I don't think this is necessary. I think you can just save `form.data`.

Reply:
Thanks @jbalboni, removed.